.coderabbit.yml

```
@@ -0,0 +1,60 @@
+language: "python"
+reviews:
+  profile: "chill"
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
+  review_status: true
+  collapse_ellipsis: false
+  auto_reply: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)